README.md

```diff
@@ -1,4 +1,12 @@
 # JobBoard
 
 This project is an open source Job Board. It's primary purpose is to serve as a
-training exercise.
\ No newline at end of file
+training exercise.
+
+
+## Getting Started
+
+1. Clone the repo `git vlone git@github.com:MattGibney/JobBoard.git`
```

Comment:
Clone spelt wrong.  The link didn't work for me either.  

I used: git clone https://github.com/MattGibney/JobBoard.git